js/exercise-00.js

```diff
@@ -0,0 +1,25 @@
+/* 
+   Create a constant variable of each of the following primitive types:
+   - String
+   - Number
+   - Boolean
+   - Null
+   - Undefined
+
+   Log out each variable to the terminal at the end.
+
+*/
+
+
+const string = "Paco";     
```

Comment:
Have you tested this script before pushing?

Is it doing what is supposed to do?

FIX: test script and fix if required.